dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -219,4 +236,60 @@ public static String getFileName(MultipartFile multipartFile)
             return multipartFile.getName();
         }
     }
+
+    private List<DSpaceObject> constructDSpaceObjectList(Context context,
+                                                         HttpServletRequest request) throws IOException {
+        List<String> list = readFromRequest(request);
+
+        List<DSpaceObject> dSpaceObjects = new LinkedList<>();
+        for (String string : list) {
+            String uuid = string.substring(string.lastIndexOf('/') + 1);
+            try {
+                for (DSpaceObjectService dSpaceObjectService : dSpaceObjectServices) {
+                    DSpaceObject dSpaceObject = dSpaceObjectService.find(context, UUIDUtils.fromString(uuid));
+                    if (dSpaceObject != null) {
+                        dSpaceObjects.add(dSpaceObject);
+                        break;
+                    }
+                }
+            } catch (SQLException e) {
+                log.error(e.getMessage(), e);
+            }
+
+        }
+        return dSpaceObjects;
+    }
+
+    private List<String> readFromRequest(HttpServletRequest request) throws IOException {
```

Comment:
This seems like an important method for uri-list requests.  We should add JavaDocs.  I *think* I understand that it's taking all the lines in the Request body and creating a List of Strings.  Is that correct?

Reply:
The JavaDocs have been included